tox.ini

```diff
@@ -31,6 +31,16 @@ basepython = python2.7
 deps = https://github.com/willkg/dennis/archive/aca57f6d.zip
 commands = dennis-cmd lint --errorsonly locale/
 
+[testenv:docker]
+commands = 
+    docker-compose -f docker-compose.yml -f scripts/docker-compose.travis.yml up -d
+    docker exec -i kuma_web_1 make compilejsi18n collectstatic
+    # TODO: mysql needs some time to set up. find a way to eliminate the waiting time
+    sleep 60
```

Comment:
Here's one solution, for example - http://brunorocha.org/python/dealing-with-linked-containers-dependency-in-docker-compose.html